web/src/features/dialog/InputDialog.tsx

```diff
@@ -104,9 +112,26 @@ const InputDialog: React.FC = () => {
                 {row.type === "input" && (
                   <Box mb={3} key={`input-${index}`} textAlign="left">
                     <Text>{row.label}</Text>
-                    <Input
-                      onChange={(e) => handleChange(e.target.value, index)}
-                    />
+                    <InputGroup>
+                      <Input
+                        onChange={(e) => handleChange(e.target.value, index)}
+                        type={!row.password || showPassword ? 'text' : 'password'}
+                      />
+                      {row.password && (
+                        <InputRightElement
+                          cursor="pointer"
+                          onClick={handleShowPassword}
+                          children={
+                            <FontAwesomeIcon
+                              fixedWidth
+                              icon={showPassword ? "eye" : "eye-slash"}
+                              fontSize="1em"
+                              style={{ paddingRight: 8 }}
+                            />
```

Comment:
Don't really like the look of the FA icon on this, maybe just a button from Chakra UI's [example](https://chakra-ui.com/docs/components/form/input#password-input-example) would be enough?

Reply:
Like this?
![image](https://user-images.githubusercontent.com/71350868/166337144-8d988115-7129-4319-9759-88d02b86e1ab.png)
![image](https://user-images.githubusercontent.com/71350868/166337159-2cee06a8-e34a-4090-991f-665cbe07cfe5.png)

Reply:
I think the icon is better, because every language has a different length of "Show" and "Hide" which would change the input text space a lot - even when switching between states.

Reply:
> I think the icon is better, because every language has a different length of "Show" and "Hide" which would change the input text space a lot - even when switching between states.

Yeah you're right.